### PR TITLE
feat(onboarding): Add alert rule configuration onboarding

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: tests/js/spec/views/settings/projectAlerts/issueEditor.spec.jsx [hunk 1]
```diff
@@ -143,5 +145,12 @@ describe('ProjectAlerts -> IssueEditor', function() {
         })
       );
     });
+
+    it('Updates the alert onobarding task', async function() {
```
Comment (billyvg) [inline tests/js/spec/views/settings/projectAlerts/issueEditor.spec.jsx]:
```suggestion
    it('updates the alert onboarding task', async function() {
```

File: src/sentry/static/sentry/app/views/settings/projectAlerts/onboardingHovercard.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+  get shouldShowHovercard() {
+    const {organization} = this.props;
+    const {dismissed} = this.state;
+
+    const hasCompletedTask = organization.onboardingTasks.find(
+      task => task.task === OnboardingTaskKey.ALERT_RULE && task.status === 'complete'
+    );
+
+    const query = qs.parse(this.props.location?.search);
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/projectAlerts/onboardingHovercard.tsx]: Why this over `location.query`?

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/projectAlerts/onboardingHovercard.tsx]: mm, good question

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/projectAlerts/onboardingHovercard.tsx]: Updated to use this, thanks.

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/components/onboardingWizard/getOnboardingTasks.tsx [hunk 3]
```diff
@@ -134,7 +134,7 @@ export default function getOnboardingTasks(
       prereq: [1],
       actionType: 'app',
       location: `/settings/${organization.slug}/projects/:projectId/alerts/`,
-      display: false,
+      display: true,
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/onboardingWizard/getOnboardingTasks.tsx]: Gonna merge with this off

**Pull request merged**